archive/schema.go

```diff
@@ -120,7 +128,11 @@ func (ark *Archive) AppendSpans(spans []SpanInfo) error {
 	return ark.Meta.Write(filepath.Join(ark.Root, metaDataFilename))
 }
 
-func OpenArchive(path string) (*Archive, error) {
+type OpenOptions struct {
+	LogFilter []string
```

Comment:
Shouldn't this be?
```suggestion
	LogFilter []LogID
```

Reply:
I'd like to use archive.LogID here, but with our current package layout, that would lead to a circular dependency.